src/api/AdminEventsRepository.js

```diff
@@ -1,10 +1,17 @@
 import { kcRequest } from "./Repository";
 
-const resource = "/admin-events";
+const resource = "/admin-events" +
+    "?operationTypes=UPDATE" +
+    "&operationTypes=CREATE" +
+    "&operationTypes=DELETE" +
+    "&resourceTypes=USER" +
+    "&resourceTypes=CLIENT_ROLE_MAPPING" +
+    "&max=100";
```

Comment:
During my meeting with Leon we agreed that it makes sense to restrict visible events to "user" events, those events that can be generated by using the User Management Console. (Note that this implies that if a user can do something in the User Management Console that _isn't_ logged, that's possibly a bug.)